src/KineticConvolution/Direction.cs

```diff
@@ -1,8 +1,8 @@
 using System;
 
-namespace KineticConvolution
+namespace Hilke.KineticConvolution
 {
-    public class Direction: IEquatable<Direction>
+    public class Direction : IEquatable<Direction>
```

Comment:
I don't think `Direction` should be a value type because `IAlgebraicNumber` might not always be a value type and it's not good practice for value types to have reference type properties.